docs/cody/clients/enable-cody-enterprise.mdx

```diff
@@ -28,6 +28,8 @@ Guardrails for public code are available to all Sourcegraph Enterprise instances
 
 It only matches code snippets that are at least **10 lines** or longer, and the search corpus **290,000** open source repositories.
 
+<Callout type="info">Currently, there is no license attribution for Guardrails, and Cody will not check any license types. It only attributes the code it finds.</Callout>
```

Comment:
I will frame it differently and combine it with the previous line item:

"Guardrails currently doesn't distinguish between different license types, instead matches any code snippets that are at least 10 lines or longer from the 290,000 indexed open source repos."